src/org/opendatakit/briefcase/util/FileSystemUtils.java

```diff
@@ -180,7 +219,7 @@ public static final boolean isODKInstancesParentFolder(File pathname) {
     return foi.exists() && foi.isDirectory();
   }
 
-
+  @SuppressWarnings("unchecked")
```

Comment:
Is it possible to parameterize the generics pathToHashMap and pathToDefinitionMap?  Could remove this warning suppression if so.